index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="vewport" content="width=device-width, initial-scale=1.0">
+    <title>Documents</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+ <boby>
+    <nav class="navbar">
+        <div class="name">Patrick & Amos</div>
+        <div class="links">
+            <a href="#about">About</a>
+            <a href="#project">Project</a>
+            <a href="#contact">Contact</a>
+        </div>
+    </nav>
+
+    <div class="main-section">
+        <section id="about">
+            <header class="header">
+                <h1>About</h1>
+                <h2>Patrick & Amos</h2>
+                <h2>Web Developer</h2>
+            </header>
+            <div class="aboutme">
+                <p>
+                    we are skilled web developer that enjoys being practical in our projects
+                </p>     
+            </div>
+                <section id="projects">
+                    <header class="header">
+                        <h1>Projects</h1>
+                        <h2>Here are some of the projects we made</h2>
+                    </header>
+                    <div class="project">
+                        <div class="project-title">Official Logo</div>
+                        <img src="images/IMG-20210410-WA0001 (1).jpg" alt="survey" class="project-img">
+                    </div>
+                    <div class="project">
+                        <div class="project-title">Logo Design</div>
+                        <img src="images/IMG-20210410-WA0002.jpg" alt="survey" class="project-img">
```

Comment:
with the images, it would be better if you were to put them in a folder of their own (called images) so you can reference them when you write the code 👍 
![200317082-5c8c4342-3595-4c2c-a821-a51c4e4eb1f7](https://user-images.githubusercontent.com/106984562/227882966-0e201c7d-8138-442f-a639-704680b4db01.jpg)

Reply:
- [ ] Same as here, could you explain why the student should follow your review(comment), is what the student did not work, or is what the student did not a better way of doing it? etc... please be more explicit in your comment.